src/post.ts

```diff
@@ -46,6 +61,31 @@ export async function post (): Promise<void> {
         return
       }
 
+      // Post Slack notification
+      if (slackToken !== '' && slackChannel !== '') {
+        try {
```

Comment:
That part feels like a good candidate to be a separate method.
This way we can reduce nesting, such as:
```js
if (slackToken && slackChannel) {
  await postSlackNotification()
}
```

However, the number of parameters to pass in would high, as always.
So it's a trade-off and up to you.

Parameters would be 👇 🙈 

```js
async function postSlackNotification({ token, channel }, { owner, repo, sha, status, actor })
```

Another way would be to define the new function inside an existing `post()` function. This way it will be able to access all variables from it, right? 🤔

Reply:
exported to a separate function